### PR TITLE
Backport PRs to branch v8.10

### PR DESCRIPTION
**Pull request by vbgl**
This is  #9961, that has been automatically closed by mistake…

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ stages:
 variables:
   # Format: $IMAGE-V$DATE [Cache is not used as of today but kept here
   # for reference]
-  CACHEKEY: "bionic_coq-V2019-03-12-V1"
+  CACHEKEY: "bionic_coq-V2019-05-02-V1"
```
Comment (ejgallego) [inline .gitlab-ci.yml]: Put a v8.10 tag so we know this image is "permanent"

**Comment by gares** [comment]
I think you also want #10392 then

**Comment by SkySkimmer** [comment]
#9855 isn't a fix, why backport it?

**Comment by ejgallego** [comment]
> #9855 isn't a fix, why backport it?

Yup, IMVHO these commits are too much of a risk for 8.10 at this stage.

**Comment by vbgl** [comment]
Thanks for watching and preventing me from doing mistakes!

**Comment by ejgallego** [comment]
> Thanks for watching and preventing me from doing mistakes!

Likely our fault for adding the fix tag somewhere, sorry for that! 

Current error handling logic is quite scary so I put the fix tag in some places, but indeed, these "bugfixes" are quite risky sometimes and hard to test.

**Pull request merged**